platforms/ios/CDVCrittercism.m

```diff
@@ -132,25 +132,25 @@ - (void) crittercismBeginTransaction:(CDVInvokedUrlCommand *)command {
         NSString* transaction = command.arguments[0];
         if ([command.arguments count] >= 2) {
           NSInteger transactionValue = [command.arguments[1] integerValue];
-          [Crittercism beginTransaction:transaction
-                                withValue:(int)transactionValue];
+          [Crittercism beginUserflow:transaction
+                           withValue:(int)transactionValue];
         } else {
-          [Crittercism beginTransaction:transaction];
+          [Crittercism beginUserflow:transaction];
         }
     }];
 }
 
 - (void) crittercismEndTransaction:(CDVInvokedUrlCommand *)command {
     [self.commandDelegate runInBackground:^{
         NSString* transaction = command.arguments[0];
-        [Crittercism endTransaction:transaction];
+        [Crittercism endUserflow:transaction];
```

Comment:
Exchanged all Crittercism.m transaction calls to their userflow counterparts.

Basically no change as [5.7.3 Crittercism.m transaction calls already automatically redirect to their userflow counterparts](https://github.com/crittercism/crittercism-ios/blob/4ee0d33ba344efa1488750e6175a7314a7f89039/src/Crittercism/Crittercism.m#L369)

This merely suppresses the easily resolved transaction deprecation messages